test/parallel/test-https-argument-of-creating.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const common = require('../common');
+
+const assert = require('assert');
+const https = require('https');
+const tls = require('tls');
+
+const dftProtocol = {};
+tls.convertNPNProtocols([ 'http/1.1' ], dftProtocol);
+
+// test for immutable `opts`
+const opts = { foo: 'bar', NPNProtocols: [ 'http/1.1' ] };
+const server1 = https.createServer(opts);
```

Comment:
It's a style we now use more often (helps frame every test case), but IMHO it's up to you...